app/utils/url_utils.rb

```diff
@@ -43,4 +43,13 @@ def extract_locale_from_url(url)
   def strip_port_from_host(host)
     host.split(":").first
   end
+
+  # Naive join method, which can be used to normalize multiple slashes
+  #
+  # Usage: URLUtils.join("foo", "bar/", "baz") => "foo/bar/baz"
+  def join(*parts)
```

Comment:
What would be the expected result for `join("foo", "/bar")`? `foo/bar` I guess?

Usually (at least in URL context), the prepending slash `/bar` means path from the URL root. E.g. if the current URL for web page is `http://example.com/foo/` and there's a link pointing to `/bar`, the result is `http://example.com/bar`.

So that's why it's not obvious what the result would be for joining path parts with prepending slashes and that's why I intentionally left it out.

What do you think, how should I proceed?